packages/form-js-viewer/src/Form.js

```diff
@@ -93,6 +93,19 @@ export default class Form {
     }
   }
 
+  destroy() {
+    this.get('eventBus').fire('form.destroy');
+
+    const container = this._container,

```

Comment:
Why not use `detach` here? It should do the same thing, right?

Reply:
Cf. https://github.com/bpmn-io/form-js/pull/78#discussion_r648149434.

Reply:
The alternative to duplication could be to be able to detach without emitting [here](https://github.com/bpmn-io/form-js/blob/master/packages/form-js-editor/src/FormEditor.js#L153).

```
this.detach(false);
```

Reply:
If we wanted to we could even move the core detach functionality into its own api, i.e expose the `noDetach` arg on a `_detach` utility only

Reply:
Yeah, I though about moving the duplicate code to a `_detach` method and using it in both places but firing the event only in `detach`. Will add a commit with a proposal.

Reply:
Done (cf. https://github.com/bpmn-io/form-js/pull/78/commits/e672778fa2169fd8fc0f3765708b02b16ac96330).

Reply:
Works for me. Works for you, too @barmac?